lib/jekyll/converters/markdown/kramdown_parser.rb

```diff
@@ -5,19 +5,20 @@ module Jekyll
   module Converters
     class Markdown
       class KramdownParser
+        attr_accessor :config
         CODERAY_DEFAULTS = {
-          "css"               => "style",
-          "bold_every"        => 10,
-          "line_numbers"      => "inline",
+          "css" => "style",
+          "bold_every" => 10,
+          "line_numbers" => "inline",
           "line_number_start" => 1,
-          "tab_width"         => 4,
-          "wrap"              => "div"
+          "wrap" => "div",
+          "tab_width" => 4
         }
 
         def initialize(config)
           Jekyll::External.require_with_graceful_fail "kramdown"
           @main_fallback_highlighter = config["highlighter"] || "rogue"
-          @config = config["kramdown"] || {}
+          @config = Marshal.load(Marshal.dump(config["kramdown"] || {}))
```

Comment:
We have been asked not to use Marshal for user data. https://github.com/jekyll/security/issues/5